doc/api/process.md

```diff
@@ -1819,6 +1819,46 @@ process.report.writeReport();
 
 Additional documentation is available in the [report documentation][].
 
+## process.resourcesUsage()
+<!-- YAML
+added: v12.x.x
+-->
+
+* Returns: {Object}
+    * `minflt` {integer}
```

Comment:
I'm not sure if it matters at the JS level, but these are all unsigned integers.